docs/kb/semgrep-appsec-platform/no-runs-in-github-merge-queues.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Learn how to work around Semgrep Managed Scans not running for pull requests in GitHub merge queues
+tags:
+  - Managed Scans
+  - GitHub
+---
+
+# Semgrep Managed Scans doesn't run for pull requests in GitHub merge queues
+
+If you:
```

Comment:
If you use Semgrep Managed Scans with GitHub Merge Queues and have marked the Semgrep scan as a required check, your merge queues may stall. Managed Scans don’t run inside merge queues, so the required Semgrep check never completes, and merges can’t proceed.